typings/__tests__/index.test.tsx

```diff
@@ -306,6 +307,18 @@ const withinGet: Array<ReactTestInstance> = [
   within(instance).getByRole('button'),
   within(instance).getByA11yState({ busy: true }),
   within(instance).getByA11yValue({ min: 10 }),
+  getQueriesForElement(instance).getByText('Test'),
```

Comment:
I wish we could do something like `within === getQueriesForElement` instead, but TS is structural and this will likely pass anyway. And I wouldn't like to resort to [hacks like described here](https://isthisit.nz/posts/2020/typescript-ii-nominal-and-structural-typing/)